radio/src/targets/common/arm/stm32/adc_driver.cpp

```diff
@@ -24,10 +24,12 @@
   // not needed
 #elif defined(PCBX10)
   const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1, -1,1,1,1,  -1};
-#elif defined(PCBX9E) && defined(HORUS_STICKS)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
 #elif defined(PCBX9E)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,1,-1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
+  #if defined(HORUS_STICKS)
```

Comment:
We typically don't indent them so that they're obvious when reading Found ourselves follwoing code too often that happened not to apply to the desired config, but the #ifdef wasn't noticed.

But again it's hard to be consistent - for example in a file like hal.h where it's nothing but defines it's nice and helpful to indent the #ifdefs - when mixed with code I don't like it for the above reason